app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -337,6 +338,7 @@ class ThreadListFragment : TwoPaneFragment(), SwipeRefreshLayout.OnRefreshListen
         }
 
         searchButton.setOnClickListener {
+            mainViewModel.isInSearch.value = true
```

Comment:
Values of `LiveDatas` shouldn't be modified directly in the `View`, here the `Fragment`.